app/Models/Enums/CourseEducationLevel.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models\Enums;
+
+enum CourseEducationLevel: string
+{
+  use EnumToArray;
+
+  case mbo = 'mbo';
+  case hbo = 'hbo';
+  case wo = 'wo';
```

Comment:
Enums moeten volledig uppercase zijn afaik.

```suggestion
  case MBO = 'mbo';
  case HBO = 'hbo';
  case WO = 'wo';
```